core/src/main/java/nl/nn/adapterframework/pipes/FixedResultPipe.java

```diff
@@ -42,15 +41,35 @@
 import nl.nn.adapterframework.util.TransformerPool;
 
 /**
- * Provides an example of a pipe. It may return the contents of a file
- * (in the classpath) when <code>filename</code> or <code>filenameSessionKey</code> is specified, otherwise the
- * input of <code>returnString</code> is returned.
+ * Produces a fixed result that does not depend on the input message. It may return the contents of a file
+ * when <code>filename</code> or <code>filenameSessionKey</code> is specified. Otherwise the
+ * value of attribute <code>returnString</code> is returned.
+ * <br/><br/>
+ * Using parameters and the attributes of this pipe, it is possible to substitute values. This pipe
+ * performs the following steps:
+ * <ol>
+ * <li>When a configuration is loaded, all expressions <code>${...}</code> within <code>Configuration.xml</code> and its referenced entities are substituted.
+ * This substitution is done according to system properties and application properties (DeploymentSpecifics.properties and others). See the Frank!Manual for details.
+ * This substitution is done before this pipe is configured or executed.
```

Comment:
Dit is iets generieks en niet iets voor deze pipe. Het zorgt hier voor onduidelijkheid.

Reply:
De tekst is weg.